### DISCUSSION TITLE
Report a bug

### DISCUSSION THREAD

**Discussion by SxinZara** [community/community#131873] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How do I report a bug or suggest a new feature?

**Comment by Sxincode** [comment]
To report a bug or suggest a new feature, follow these steps:

Check existing issues: Before reporting a new bug or suggesting a feature, check the existing issues to see if it has already been reported or suggested.
Create a new issue: If you don't find any related issues, go to the 'Issues' tab in the repository and click 'New issue.'
Choose the issue type: Select whether you are reporting a bug or suggesting a feature.
Provide a descriptive title: Write a clear and concise title for your issue.
Describe the issue: For bug reports, describe the steps to reproduce the bug, expected behavior, actual behavior, and any relevant screenshots or error messages. For feature suggestions, describe the feature, its benefits, and any relevant examples or use cases.
Submit the issue: Once you have filled in the details, click 'Submit new issue' to create the issue.
Follow up: Engage in any discussions or follow-ups on the issue. Respond to questions or provide additional information if needed.

**Reply by ebndev** [reply]
Hi @Sxincode, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
